packages/nc-gui/components/smartsheet/column/FormulaOptions.vue

```diff
@@ -238,7 +242,7 @@ function validateAgainstMeta(parsedTree: any, errors = new Set(), typeErrors = n
     errors = new Set([...errors, ...typeErrors])
   } else if (parsedTree.type === JSEPNode.IDENTIFIER) {
     if (
-      columns.value
+      supportedColumns.value
         .filter((c: Record<string, any>) => !column || column.value?.id !== c.id)
         .every((c: Record<string, any>) => c.title !== parsedTree.name)
```

Comment:
Do we need the `Record<string, any>` cast? Is `ColumnType` not auto-inferred? If not, is `ColumnType` not ok as a type?

Further down a similar code block uses `c: ColumnType`